ras/rtas_dbg.py

```diff
@@ -20,19 +20,22 @@
 from avocado import skipIf
 
 IS_POWER_NV = 'PowerNV' in genio.read_file('/proc/cpuinfo').rstrip('\t\r\n\0')
+IS_KVM_GUEST = 'qemu' in open('/proc/cpuinfo', 'r').read()
 
 
 class rtas_dbg(Test):
 
-    @skipIf(IS_POWER_NV, "This test is supported on PowerVM environment")
+    @skipIf(IS_POWER_NV or IS_KVM_GUEST,
+            "This test is supported only on PowerVM environment")
     def setUp(self):
         sm = SoftwareManager()
         if not sm.check_installed("powerpc-utils") and \
            not sm.install("powerpc-utils"):
             self.cancel("Fail to install required 'powerpc-utils' package")
 
     def test_rtas_dbg(self):
-        lists = self.params.get('list', default=['-l', '-l 14', '-l get-power-level'])
+        lists = self.params.get('list', default=['-l', '-l 14',
+                                                 '-l get-power-level'])
```

Comment:
can we have this style issue fix in another commit, please

Reply:
Sure. Have updated the code accordingly.